src/components/ResourcePreview/index.jsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-// import { DataTable } from '@civicactions/data-catalog-components';
+import { DataTable } from '@civicactions/data-catalog-components';
```

Comment:
This is is also being worked on in another branch and can be left commented out. Once I finish that update I can add it back to the different pages tables appear on.